yt/yt/client/api/operation_client.cpp

```diff
@@ -227,5 +227,56 @@ void TListOperationsAccessFilter::Register(TRegistrar registrar)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+TGetJobStderrResponse TGetJobStderrResponse::MakeJobStderr(const TSharedRef& data, const TGetJobStderrOptions& options)
+{
+    auto totalSize = std::ssize(data);
+    auto endOffset = totalSize;
+    auto offset = options.Offset.value_or(0);
+    auto limit = options.Limit.value_or(0);
+
+    if (!offset && !limit) {
+        return {
+            .Data = data,
+            .TotalSize = totalSize,
+            .EndOffset = endOffset,
+        };
+    };
+
+    size_t firstPos = 0;
+    if (offset >= 0) {
+        firstPos = offset;
+    } else if (-offset > static_cast<i64>(data.size())) {
+        firstPos = 0;
+    } else {
+        firstPos = data.size() + offset;
+    }
```

Comment:
Do we still support negative offsets? Why do we want to introduce inconsistency between read-file and get-stderr?